src/main/java/com/study/cheeper/controller/HomeController.java

```diff
@@ -1,15 +1,31 @@
 package com.study.cheeper.controller;
 
+import com.study.cheeper.model.User;
+import com.study.cheeper.repository.CheepRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.context.SecurityContextHolder;
 import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.ModelAndView;
 
 @Controller
 public class HomeController {
 
+    @Autowired
+    private CheepRepository cheepRepository;
+
     @GetMapping("/home")
-    public String home() { return "/home"; }
+    public ModelAndView home() {
```

Comment:
Olhando para o método `defaultHome()`, se for só pelos diferentes paths, o `GetMapping` recebe um array, ai você poderia ter só um método:

```suggestion
@GetMapping(value={"/", "/home"})
public ModelAndView home() {
```